frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboards.ts

```diff
@@ -205,6 +205,18 @@ export type DashboardsOverviewPrometheusActivity = ExtensionDeclaration<
   }
 >;
 
+/** Adds an item to the Details card of Overview Dashboard */
+export type DashboardsOverviewDetailItem = ExtensionDeclaration<
```

Comment:
we need to add example to demo plugin https://github.com/openshift/console/tree/master/dynamic-demo-plugin

Reply:
can you please also rename it to 
```suggestion
export type ClusterOverviewDetailItem = ExtensionDeclaration<
```

and add it to https://github.com/openshift/console/blob/9e695b542a6613101b6778ebf214a69c72df80ff/frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-overview.ts instead of this file. Im trying to make it clear that the extension is for Cluster Overview page.

Reply:
done